Formula/y/yara.rb

```diff
@@ -7,6 +7,11 @@ class Yara < Formula
   revision 1
   head "https://github.com/VirusTotal/yara.git", branch: "master"
 
+  livecheck do
```

Comment:
```suggestion
  # Upstream sometimes creates releases that use a stable tag (e.g., `v1.2.3`)
  # but are labeled as "pre-release" on GitHub, so it's necessary to use the
  # `GithubLatest` strategy.
  livecheck do
```

The `stable` URL isn't a GitHub release asset, so we should explain why we're using the `GithubLatest` strategy.